src/Functional/Repeat.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright (C) 2011-2017 by Lars Strojny <lstrojny@php.net>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+namespace Functional;
+
+use Closure;
+use Functional\Exceptions\InvalidArgumentException;
+
+/**
+ * Creates a function that can be used to repeat the execution of $callback.
+ *
+ * @param callable $callback
+ *
+ * @return Closure
+ */
+function repeat(callable $callback) {
+    return function ($times) use ($callback) {
+        InvalidArgumentException::assertPositiveInteger($times, __FUNCTION__, 1);
+
+        for ($i = 0; $i < $times; $i++) {
+            call_user_func($callback);
```

Comment:
`$callback()` would work

Reply:
Isn't `$callback()` PHP7 only?

Reply:
I think you can assign a function to a variable [since PHP5.3](http://sandbox.onlinephpfunctions.com/code/bd8d67be8c02818d72a0df2420d3cd53b89fa7ef) 😄

Reply:
@lstrojny Done.